portfolio/src/main/webapp/script.js

```diff
@@ -12,23 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
-
 function getComments() {
-    fetch('/data').then(response => response.text()).then((myString) => {
-        document.getElementById('comments').innerText = myString;
+    fetch('/data').then(response => response.json()).then((json) => {
+        
+        const commentsElement = document.getElementById('comments');
+        commentsElement.innerText = '';
+        const comments = json.comments;
+        console.log('Received: ' + comments);
+        for (let i = 0; i < comments.length; i++) {
+            commentsElement.appendChild(
+                createListElement(comments[i].userName + ': ' + comments[i].comment));
```

Comment:
The formatting for this block seems off. Use 2 spaces. You can also use the clang-format-path to do the formatting

Reply:
Done: d8b8c5f